### ISSUE TITLE
Horizontal alignment does not appear to work in OpenOffice or LibreOffice

### ISSUE THREAD

**Issue by CyberBotX** [EPPlusSoftware/EPPlus#380]
I have found that if I try to set horizontal alignment on either cells or columns with EPPlus, while the XLSX will have the correct alignments within Excel (at least when tested via Excel Mobile from the Microsoft Store), it does not with OpenOffice or LibreOffice. This seems like it might be a regression, as demonstrated below.

The following is a way to create the files I am testing with, which can be run via LINQPad 6 after adding EPPlus via F4:
```
foreach (bool onColumns in new[] { true, false })
{
	var ep = new ExcelPackage();
	var ws = ep.Workbook.Worksheets.Add("Test");

	// Header area (along with freezing the header in the view)
	ws.Cells[1, 1, 1, 8].Style.Font.Bold = true;
	for (int i = 1; i < 9; ++i)
		ws.Cells[1, i].Value = $"Test {i}";
	ws.View.FreezePanes(2, 1);

	if (onColumns)
	{
		// Set the horizontal alignment on the columns themselves
		ws.Column(3).Style.HorizontalAlignment = ws.Column(4).Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Left;
		ws.Column(5).Style.HorizontalAlignment = ws.Column(6).Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Center;
		ws.Column(7).Style.HorizontalAlignment = ws.Column(8).Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Right;
	}
	else
	{
		// Set the horizontal alignment on the cells of the header
		ws.Cells[1, 3, 1, 4].Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Left;
		ws.Cells[1, 5, 1, 6].Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Center;
		ws.Cells[1, 7, 1, 8].Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Right;
	}

	for (int row = 2; row < 30; ++row)
	{
		for (int i = 1; i < 9; ++i)
			ws.Cells[row, i].Value = row % 2 == 0 ? 8 * (row - 2) + i : $"Test {8 * (row - 2) + i}";
		if (!onColumns)
		{
			// Set the horizontal alignment on this row's cells
			ws.Cells[row, 3, row, 4].Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Left;
			ws.Cells[row, 5, row, 6].Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Center;
			ws.Cells[row, 7, row, 8].Style.HorizontalAlignment = OfficeOpenXml.Style.ExcelHorizontalAlignment.Right;
		}
	}

	ws.Cells.AutoFitColumns(0);

	ep.SaveAs(new FileInfo(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), $"AlignmentTest-On{(onColumns ? "Columns" : "Cells")}.xlsx")));
}
```

When I have `onColumns` set to true, it sets the horizontal alignment on the columns themselves. When it is false, it sets the horizontal alignment on the cells themselves. Running the above code in LINQPad should generate both files to test. (Alternatively, the code can be put inside the body of the Main method of a C# console application.)

The alignment seems to be working in 5.6.2 but not in 5.6.3. In OpenOffice none of the alignments are recognized, and in LibreOffice the same happens when the alignments are set on the cells themselves, while when set on the columns themselves, the first row isn't aligned while the rest of the cells in the column are.

Although, I am not 100% sure if it is working completely as expected in 5.6.2 either. In the above test, yes the alignments work in 5.6.2 and don't work completely in 5.6.3, but in my actual code (which is way too complex to show here), I get a weirder issue where I set the horizontal alignment to Center on multiple columns in a for loop and only the last of the columns has Center alignment (but the first row does not) and the other columns do not have Center alignment.

**Comment by JanKallman** [comment]
Thanks, I'll have a look

**Comment by JanKallman** [comment]
Try this fix using our develop nuget feed ... https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed.
The applyAlignment element was not correctly set when applying the style. I have verified this is LibreOffice 7.1.2.2.

**Comment by CyberBotX** [comment]
Sorry for the delayed response to this, I was doing some more thorough testing to see if any other versions of EPPlus 5 were affected by this. (I found some versions of EPPlus 4 were affected but that is irrelevant to this issue.) It appears as though it was indeed 5.6.3 that introduced the bug, and testing 5.6.3.589-20210427-develop from the EPPlus develop NuGet feed does appear to do things properly again.

**Comment by JanKallman** [comment]
Thanks for the feedback

**Comment by JanKallman** [comment]
Fixed in 5.6.4
